apps/docs/src/app/core/core-documentation.module.ts

```diff
@@ -16,7 +16,6 @@ import {
     SELECT_MOBILE_CONFIG
 } from '../documentation/utilities/consts';
 
-
```

Comment:
remove file

Reply:
Could you be more specific what is the problem?